docs/howtos/runlocalmultiplayeragent.md

```diff
@@ -15,7 +15,7 @@ To run LocalMultiplayerAgent to test your game servers for Thundernetes, you'll
 
 - Download latest version of LocalMultiplayerAgent from the [Releases](https://github.com/PlayFab/MpsAgent/releases) page on GitHub
 - [Install Docker Desktop on Windows](https://docs.docker.com/docker-for-windows/install/)
-- Make sure it's running on [Linux Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers)
+- Make sure it's running on [Linux Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers) if your game server is built for the Linux operating system
```

Comment:
would recommeend something like
"Make sure it's running on the appropriate container for the operating system your server needs (either linux containers (with a link) or windows containers (with another link))